app/controllers/entries/likes_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Entries
+  class LikesController < ApplicationController
+    before_action :set_entry
+    before_action :set_like, only: :destroy
+
+    def create
+      like = Entry::Like.build(entry: @entry, user: Current.user)
```

Comment:
Se adicionarmos a variável de instância, podemos checar se o like foi criado ou destruído quando renderizarmos o botão de like `@like.persisted?`:

```suggestion
      @like = Entry::Like.build(entry: @entry, user: Current.user)
```

Reply:
massa, melhor do que fazer a query